modules/document_repository/php/document_repository.class.inc

```diff
@@ -233,7 +233,7 @@ class Document_Repository extends \NDB_Menu_Filter
     /**
      * ParseCategory function
      *
-     * @param string $value the value
+     * @param array $value the value
      *
      * @return array
```

Comment:
It look like the return value is a string
Plus, the DB query is not sanitized. Should be change to pselect (in another PR).